unmock/core/http.py

```diff
@@ -29,11 +22,9 @@ def initialize(unmock_options: Optional[UnmockOptions] = None, story: Optional[L
     :param story: An optional list of unmock stories to initialize the state. These represent previous calls to unmock
         and make unmock stateful.
     :type story List[str]
-    :param token: An optional unmock token identifying your account.
-    :type token str
+    :param refresh_token: An optional unmock *refresh token* identifying your account.
+    :type refresh_token str
```

Comment:
Should this be `:type refresh_token: str, optional` (including the second colon)?